app/views/ShareListView/index.js

```diff
@@ -22,31 +21,27 @@ import {
 	closeSearchHeader as closeSearchHeaderAction
 } from '../../actions/rooms';
 import DirectoryItem, { ROW_HEIGHT } from '../../presentation/DirectoryItem';
-import ServerItem, { ROW_HEIGHT as ROW_HEIGHT_SERVER } from '../../presentation/ServerItem';
+import ServerItem from '../../presentation/ServerItem';
 import { CloseShareExtensionButton, CustomHeaderButtons, Item } from '../../containers/HeaderButton';
-import SearchBar from '../RoomsListView/ListHeader/SearchBar';
+import SearchBar from './SearchBar';
 import ShareListHeader from './Header';
 
 import styles from './styles';
 
 const SCROLL_OFFSET = 56;
```

Comment:
It never hides.
Can you place the searchbox at the header?

Reply:
done.